marklogic-data-hub-spark-connector/build.gradle

```diff
@@ -82,22 +82,30 @@ task copyModulesForConnectorFromCoreProject(type: Copy) {
 processResources.dependsOn copyModulesForConnectorFromCoreProject
 
 shadowJar {
-    classifier "all"
+    classifier "pre"
     // Relocate packages of classes that the ML Java Client needs; it can't use the versions provided by Spark
     relocate('okio', 'marklogic.okio')
 
     // Relocating this package from Jackson. Relocating all of com.fasterxml.jackson results in the Scala
     // JacksonParser class not working, as it receives the wrong version of JsonFactory via our JsonRowParser class.
     // Fortunately, only relocating this package avoids that error and still results in the correct Jackson classes
     // being used by the ML Java Client and by Spark itself.
-    relocate('com.fasterxml.jackson.databind', 'marklogic.com.fasterxml.jackson.databind')
+    relocate('com.fasterxml.jackson', 'marklogic.com.fasterxml.jackson')
 }
 build.dependsOn shadowJar
-
+task fixShadowJar(type: Jar, dependsOn: "shadowJar") {
```

Comment:
This task needs a description to explain what it's doing here - i.e. that the relocated JsonRowParser class is being replaced with the original version so that it still uses the Spark version of Jackson as opposed to the ML version of Jackson.